config.go

```diff
@@ -18,11 +18,6 @@ type Config struct {
 	// a leader before we attempt an election.
 	ElectionTimeout time.Duration
 
-	// CommitTimeout controls the time without an Apply() operation
```

Comment:
I think this was here before so that Follower's commitIndexes would be updated in the absence of new entries to replicate. I still have no clue why it was randomized.

Now, commitIndex should be sent as part of every heartbeat. Is it important to send out new commitIndexes before the next heartbeat? We could accelerate the next heartbeat to achieve that goal, and I could see needing a config option for it, but let's name and document it clearly.